platform/ui/src/components/studyBrowser/Thumbnail.js

```diff
@@ -11,7 +14,22 @@ function ThumbnailFooter({
   SeriesNumber,
   InstanceNumber,
   numImageFrames,
+  hasWarnings
```

Comment:
Lets add proptype for this one, just to clarify that its a promise

Reply:
we have it already in line https://github.com/OHIF/Viewers/pull/2331/files#diff-bcda928687ffc00e7bf6a528eec5d9f90f09501ec4791cbe67450eb012c398d8R200 . Did you mean something else or add docs?

Reply:
Missed it, thanks